.github/chainguard/dd-trace-java.release.sts.yaml

```
@@ -0,0 +1,12 @@
+issuer: https://gitlab.ddbuild.io
+
+# DEBUG: ref should be "master"
+subject: "project_path:DataDog/apm-reliability/dd-trace-java:ref_type:branch:ref:sarahchen6/use-octo-sts"
+
+claim_pattern:
+  project_path: "DataDog/apm-reliability/dd-trace-java"
+  ref_type: "branch"
+  # ref: "master"
```

this will not work for tags, see [this test](https://gitlab.ddbuild.io/DataDog/dd-octo-sts-sandbox/-/jobs/1033180338) run for similar claims.
```suggestion
subject_pattern: "project_path:DataDog/apm-reliability/dd-trace-java:ref_type:tag:ref:v.*"

claim_pattern:
  project_path: "DataDog/apm-reliability/dd-trace-java"
  ref_type: "tag"
  ref: "v.*"
```